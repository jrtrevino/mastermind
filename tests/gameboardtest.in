435435
F
3
20
AAA
