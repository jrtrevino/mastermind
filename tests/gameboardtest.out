Enter the seed for random number generation: Enter the maximum letter for the game (A-Z): Enter the number of positions for the game (1-8): Enter the number of guesses allowed for the game: Initialized Game Board: DEE

Enter guess 1: Nope, 0 exact guesses and 0 inexact guesses

Enter guess 2: 